crates/jstz_cli/src/bridge/withdraw.rs

```diff
@@ -0,0 +1,39 @@
+use crate::{
+    bridge::convert_tez_to_mutez,
+    config::{Config, NetworkName},
+    error::{bail_user_error, Result},
+    run,
+    term::styles,
+    utils::AddressOrAlias,
+};
+use log::debug;
+
+pub async fn exec(
+    to: AddressOrAlias,
+    amount: f64,
+    network: Option<NetworkName>,
+) -> Result<()> {
+    let cfg = Config::load()?;
+
+    // Check network
+    if cfg.network_name(&network)? == NetworkName::Dev && cfg.sandbox.is_none() {
+        bail_user_error!(
+            "No sandbox is currently running. Please run {}.",
+            styles::command("jstz sandbox start")
+        );
+    }
```

Comment:
I assume this was copied from deposit, but this is actually a bug. Withdrawals are/should be supported on more than just the sandbox network

Reply:
Ah yeah. Isn't that true for deposit as well?

Reply:
This checks that the sandbox is running if the network is set to dev which is correct I think?

Reply:
I think we want to check if a network is set, not whether the network is the sandbox. We only did it for deposit because we relied on the local `jstz_bridge` alias for depositing

Reply:
Oh I see. I always thought it was to improve the UX since if `dev` is set, then the cli could provide a very helpful message to start the sandbox.

Reply:
We should have a check for a network in the future w/ a helpful message, but it shouldn't only check for `dev`

Reply:
I suggest we fix this in a later PR, the issue is tracked here: https://linear.app/tezos/issue/JSTZ-74/support-bridge-commands-for-remote-networks